source/content.css

```diff
@@ -274,7 +274,7 @@ followed someone
 }
 
 .news .body > .border-gray-light {
-	border: none !important; /* Remove border between dashboard news items */
+	border: unset !important; /* Remove border between dashboard news items */
```

Comment:
Why?

Without this it can be merged

Reply:
`npm test` complained about having `none` in there… :<

Reply:
@bfred-it 
![image](https://user-images.githubusercontent.com/11782/37463967-2c4c7594-2857-11e8-84bf-5281fe626e44.png)

Reply:
You might need to update your local install, this was [fixed recently in stylelint/xo](https://github.com/xojs/stylelint-config-xo/commit/1819f8441e30a8f3d4789f277e67207f56c61df1) and [set to none](https://github.com/sindresorhus/refined-github/commit/18fe3cb10d79f6d0ca84f3c445c62e6b3ac2e087) here in RGH

Reply:
Oh my… 🙈 Will do it and remove the unnecessary commit. 👍